### PR TITLE
Fixes: Animate text #220

### PR DESCRIPTION
**Pull request by Dimple-Choudhary**
## Changes proposed

Animate Heading tags with the type effect like a typewriter

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/Components/Herosection/index.js [hunk 1]
```diff
@@ -1,36 +1,64 @@
-import React, { useState } from 'react'
-import Video from '../../videos/video1.mp4'
-import { HeroContainer, HeroBg, VideoBg, HeroContent, HeroBtnWrapper, HeroH1, HeroP, ArrowForward, ArrowRight } from './HeroElements'
-import { Button } from '../ButtonElements'
-import {Link} from "react-router-dom";
+
+import React, { useState, useEffect, useRef } from 'react';
+import Typed from 'typed.js';
```
Comment (MonalikaPatnaik) [inline src/Components/Herosection/index.js:3]: Hey...where is this typed.js?

File: src/Components/InfoSection/index.js [hunk 2]
```diff
@@ -1,24 +1,45 @@
-import React from 'react';
+import React, {  useEffect, useRef } from 'react';
+import Typed from 'typed.js';
```
Comment (MonalikaPatnaik) [inline src/Components/InfoSection/index.js:2]: Here also!

File: src/Components/Services/index.js [hunk 3]
```diff
@@ -1,15 +1,31 @@
-import React from 'react'
+import React, {  useEffect, useRef } from 'react';
+import Typed from 'typed.js';
```
Comment (MonalikaPatnaik) [inline src/Components/Services/index.js:2]: Here also..

**Comment by Dimple-Choudhary** [comment]
Typed.js is a JavaScript library,It is installed by npm.
The code uses the useEffect hook to create a Typed.js typing animation with specific options.
Screenshot (54)

**Comment by MonalikaPatnaik** [comment]
> Typed.js is a JavaScript library,It is installed by npm. The code uses the useEffect hook to create a Typed.js typing animation with specific options. Screenshot (54)

Okay, seems that I need to run npm install typed.js :-)

**Pull request merged**